[2022-08-23 09:15:39,060] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 09:15:45,561] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 6021
[2022-08-23 09:15:45,744] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-08-23 09:15:45,744] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 9499
[2022-08-23 09:15:59,185] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addnews/ HTTP/1.1" 200 639
[2022-08-23 09:15:59,215] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:15:59,216] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:15:59,216] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:16:15,292] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 9075
[2022-08-23 09:18:04,215] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 2
[2022-08-23 09:18:04,449] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 2
[2022-08-23 09:18:19,787] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /newsPosts/ HTTP/1.1" 201 9239
[2022-08-23 09:19:38,667] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /newsPosts/ HTTP/1.1" 201 9239
[2022-08-23 09:20:12,528] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /newsPosts/ HTTP/1.1" 201 9253
[2022-08-23 09:20:15,440] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 9607
[2022-08-23 09:20:31,120] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news HTTP/1.1" 301 0
[2022-08-23 09:20:31,382] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /news/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 724, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 696, in get_context
    'post_form': self.get_rendered_html_form(data, view, 'POST', request),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 511, in get_rendered_html_form
    return self.render_form_for_serializer(serializer)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 518, in render_form_for_serializer
    return form_renderer.render(
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 372, in render
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\library.py", line 192, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\templatetags\rest_framework.py", line 87, in render_field
    return renderer.render_field(field, style)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 351, in render_field
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 166, in render
    values = self.sequence.resolve(context, ignore_failures=True)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 671, in resolve
    obj = self.var.resolve(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 796, in resolve
    value = self._resolve_lookup(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 858, in _resolve_lookup
    current = current()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 220, in iter_options
    self.get_choices(cutoff=self.html_cutoff),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 202, in get_choices
    return OrderedDict([
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 205, in <listcomp>
    self.display_value(item)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 226, in display_value
    return str(instance)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\models.py", line 163, in __str__
    return self.name3
AttributeError: 'NewsPosts' object has no attribute 'name3'
[2022-08-23 09:20:31,390] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /news/ HTTP/1.1" 500 250142
[2022-08-23 09:20:48,083] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addnews/ HTTP/1.1" 200 639
[2022-08-23 09:20:48,112] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:20:48,112] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:20:48,113] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:21:14,975] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addpark/ HTTP/1.1" 200 639
[2022-08-23 09:21:15,031] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:21:15,034] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:21:15,034] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:21:22,145] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /addpark/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\mysql\base.py", line 73, in execute
    return self.cursor.execute(query, args)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\cursors.py", line 148, in execute
    result = self._query(query)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\cursors.py", line 310, in _query
    conn.query(q)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 775, in _read_query_result
    result.read()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 1156, in read
    first_packet = self.connection._read_packet()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.DataError: (1406, "Data too long for column 'introduceMP3EN' at row 1")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py", line 156, in add_park
    Park.objects.create(naturalLevel=naturalLevel_v, ParkLevel=ParkLevel_v,
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\query.py", line 453, in create
    obj.save(force_insert=True, using=self.db)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\base.py", line 739, in save
    self.save_base(using=using, force_insert=force_insert,
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\base.py", line 776, in save_base
    updated = self._save_table(
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\base.py", line 881, in _save_table
    results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\base.py", line 919, in _do_insert
    return manager._insert(
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\query.py", line 1270, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\sql\compiler.py", line 1416, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\utils.py", line 98, in execute
    return super().execute(sql, params)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\backends\mysql\base.py", line 73, in execute
    return self.cursor.execute(query, args)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\cursors.py", line 148, in execute
    result = self._query(query)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\cursors.py", line 310, in _query
    conn.query(q)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 775, in _read_query_result
    result.read()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 1156, in read
    first_packet = self.connection._read_packet()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\pymysql\err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.DataError: (1406, "Data too long for column 'introduceMP3EN' at row 1")
[2022-08-23 09:21:22,159] [basehttp.py:161] [basehttp:log_message] [ERROR]- "POST /addpark/ HTTP/1.1" 500 332589
[2022-08-23 09:25:18,584] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addpark/ HTTP/1.1" 200 639
[2022-08-23 09:25:18,611] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:25:18,611] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:25:18,612] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:25:26,199] [basehttp.py:161] [basehttp:log_message] [INFO]- "POST /addpark/ HTTP/1.1" 200 639
[2022-08-23 09:25:26,229] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:25:26,230] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:25:26,232] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:25:31,149] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addnews/ HTTP/1.1" 200 639
[2022-08-23 09:25:31,187] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:25:31,187] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:25:31,187] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:27:24,563] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 09:27:25,714] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 09:27:33,833] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 09:27:39,240] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addnews/ HTTP/1.1" 200 639
[2022-08-23 09:27:39,269] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:27:39,269] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:27:39,270] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:28:03,283] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /addnews/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py", line 196, in add_news
    fatherParkID= NewsPosts.objects.get(value=row[8].value)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\query.py", line 435, in get
    raise self.model.DoesNotExist(
polls.models.NewsPosts.DoesNotExist: NewsPosts matching query does not exist.
[2022-08-23 09:28:03,287] [basehttp.py:161] [basehttp:log_message] [ERROR]- "POST /addnews/ HTTP/1.1" 500 84141
[2022-08-23 09:28:42,799] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 09:28:43,688] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 09:30:19,164] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 09:30:22,162] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 6021
[2022-08-23 09:30:26,614] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /addnews/ HTTP/1.1" 200 639
[2022-08-23 09:30:26,644] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/bootstrap.min.js HTTP/1.1" 404 1816
[2022-08-23 09:30:26,644] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/js/jquery-3.6.0.min.js HTTP/1.1" 404 1825
[2022-08-23 09:30:26,645] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /static/css/bootstrap.min.css HTTP/1.1" 404 1822
[2022-08-23 09:30:46,014] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /addnews/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py", line 196, in add_news
    fatherParkID= NewsPosts.objects.get(id=row[8].value)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\db\models\query.py", line 435, in get
    raise self.model.DoesNotExist(
polls.models.NewsPosts.DoesNotExist: NewsPosts matching query does not exist.
[2022-08-23 09:30:46,018] [basehttp.py:161] [basehttp:log_message] [ERROR]- "POST /addnews/ HTTP/1.1" 500 84128
[2022-08-23 09:33:05,318] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:33:05,448] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:33:26,469] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:33:26,535] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:35:21,098] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:35:37,025] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:35:47,423] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:37:20,719] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:38:02,869] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:38:03,201] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:39:02,110] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:39:02,257] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:39:59,402] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:39:59,620] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:40:23,676] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:40:23,902] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:40:44,343] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:40:44,408] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:40:55,801] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:40:55,891] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:41:30,689] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:41:30,839] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:41:46,914] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:41:47,105] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:42:46,403] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:42:46,516] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:43:23,585] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:43:23,652] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:43:44,277] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:43:44,387] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:43:58,640] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:43:58,855] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:45:34,885] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:45:47,529] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:45:47,682] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:46:14,159] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:46:14,363] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:46:32,823] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:46:33,015] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 09:46:50,922] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 09:46:51,071] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:09:35,911] [log.py:224] [log:log_response] [WARNING]- Not Found: /News/
[2022-08-23 10:09:35,912] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /News/ HTTP/1.1" 404 9481
[2022-08-23 10:09:35,940] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:09:36,085] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:40:20,526] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 10:41:22,783] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 10:41:34,575] [log.py:224] [log:log_response] [WARNING]- Not Found: /News/
[2022-08-23 10:41:34,576] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /News/ HTTP/1.1" 404 9481
[2022-08-23 10:41:34,642] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:41:34,728] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:45:23,821] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:46:36,121] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:46:36,256] [log.py:224] [log:log_response] [WARNING]- Not Found: /News/
[2022-08-23 10:46:36,257] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /News/ HTTP/1.1" 404 9481
[2022-08-23 10:46:36,385] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:47:01,746] [log.py:224] [log:log_response] [WARNING]- Not Found: /News/
[2022-08-23 10:47:01,747] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /News/ HTTP/1.1" 404 9481
[2022-08-23 10:47:01,837] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-08-23 10:47:01,838] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 9499
[2022-08-23 10:47:09,589] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 6021
[2022-08-23 10:47:09,611] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
[2022-08-23 10:47:09,614] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
[2022-08-23 10:47:09,616] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
[2022-08-23 10:47:09,617] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1719
[2022-08-23 10:47:09,618] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 200 89476
[2022-08-23 10:47:09,618] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
[2022-08-23 10:47:09,620] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
[2022-08-23 10:47:09,623] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
[2022-08-23 10:47:09,631] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
[2022-08-23 10:47:09,631] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
[2022-08-23 10:47:09,641] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
[2022-08-23 10:47:13,559] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /news/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 724, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 696, in get_context
    'post_form': self.get_rendered_html_form(data, view, 'POST', request),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 511, in get_rendered_html_form
    return self.render_form_for_serializer(serializer)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 518, in render_form_for_serializer
    return form_renderer.render(
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 372, in render
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\library.py", line 192, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\templatetags\rest_framework.py", line 87, in render_field
    return renderer.render_field(field, style)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 351, in render_field
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 166, in render
    values = self.sequence.resolve(context, ignore_failures=True)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 671, in resolve
    obj = self.var.resolve(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 796, in resolve
    value = self._resolve_lookup(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 858, in _resolve_lookup
    current = current()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 220, in iter_options
    self.get_choices(cutoff=self.html_cutoff),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 202, in get_choices
    return OrderedDict([
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 205, in <listcomp>
    self.display_value(item)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 226, in display_value
    return str(instance)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\models.py", line 163, in __str__
    return self.name3
AttributeError: 'NewsPosts' object has no attribute 'name3'
[2022-08-23 10:47:13,573] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /news/ HTTP/1.1" 500 282613
[2022-08-23 10:47:21,948] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /news/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 724, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 696, in get_context
    'post_form': self.get_rendered_html_form(data, view, 'POST', request),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 511, in get_rendered_html_form
    return self.render_form_for_serializer(serializer)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 518, in render_form_for_serializer
    return form_renderer.render(
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 372, in render
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\library.py", line 192, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\templatetags\rest_framework.py", line 87, in render_field
    return renderer.render_field(field, style)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 351, in render_field
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 166, in render
    values = self.sequence.resolve(context, ignore_failures=True)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 671, in resolve
    obj = self.var.resolve(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 796, in resolve
    value = self._resolve_lookup(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 858, in _resolve_lookup
    current = current()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 220, in iter_options
    self.get_choices(cutoff=self.html_cutoff),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 202, in get_choices
    return OrderedDict([
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 205, in <listcomp>
    self.display_value(item)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 226, in display_value
    return str(instance)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\models.py", line 163, in __str__
    return self.name3
AttributeError: 'NewsPosts' object has no attribute 'name3'
[2022-08-23 10:47:21,958] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /news/ HTTP/1.1" 500 282613
[2022-08-23 10:47:25,721] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 237923
[2022-08-23 10:47:26,119] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /static/rest_framework/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 200 18028
[2022-08-23 10:47:29,808] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /news/
Traceback (most recent call last):
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 724, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 696, in get_context
    'post_form': self.get_rendered_html_form(data, view, 'POST', request),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 511, in get_rendered_html_form
    return self.render_form_for_serializer(serializer)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 518, in render_form_for_serializer
    return form_renderer.render(
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 372, in render
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\library.py", line 192, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\templatetags\rest_framework.py", line 87, in render_field
    return renderer.render_field(field, style)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\renderers.py", line 351, in render_field
    return template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\defaulttags.py", line 166, in render
    values = self.sequence.resolve(context, ignore_failures=True)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 671, in resolve
    obj = self.var.resolve(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 796, in resolve
    value = self._resolve_lookup(context)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\django\template\base.py", line 858, in _resolve_lookup
    current = current()
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 220, in iter_options
    self.get_choices(cutoff=self.html_cutoff),
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 202, in get_choices
    return OrderedDict([
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 205, in <listcomp>
    self.display_value(item)
  File "C:\Users\young\AppData\Local\Programs\Python\Python38\lib\site-packages\rest_framework\relations.py", line 226, in display_value
    return str(instance)
  File "E:\zl\git-django-qyzs\django_wx_qhzs\polls\models.py", line 163, in __str__
    return self.name3
AttributeError: 'NewsPosts' object has no attribute 'name3'
[2022-08-23 10:47:29,817] [basehttp.py:161] [basehttp:log_message] [ERROR]- "GET /news/ HTTP/1.1" 500 282613
[2022-08-23 10:48:39,905] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\models.py changed, reloading.
[2022-08-23 10:48:40,834] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 10:49:00,638] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 10:49:06,360] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/ HTTP/1.1" 200 187670
[2022-08-23 10:50:11,886] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:50:11,915] [log.py:224] [log:log_response] [WARNING]- Not Found: /News/
[2022-08-23 10:50:11,917] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /News/ HTTP/1.1" 404 9481
[2022-08-23 10:50:12,071] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:50:33,489] [log.py:224] [log:log_response] [WARNING]- Not Found: /News/
[2022-08-23 10:50:33,503] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /News/ HTTP/1.1" 404 9481
[2022-08-23 10:50:33,504] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:50:33,646] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:51:19,270] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:51:19,449] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:51:19,529] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/ HTTP/1.1" 200 110505
[2022-08-23 10:54:27,670] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:54:27,877] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:54:27,909] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-id&limit=20&offset=0 HTTP/1.1" 200 8974
[2022-08-23 10:55:05,117] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:55:05,264] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:55:05,300] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-id&limit=20&offset=0 HTTP/1.1" 200 8974
[2022-08-23 10:56:02,460] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:56:02,709] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:56:02,728] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-id&limit=20&offset=0 HTTP/1.1" 200 8974
[2022-08-23 10:58:27,147] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 10:58:27,391] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 10:58:27,436] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:02:40,816] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:02:40,868] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:02:40,942] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:03:34,305] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:03:34,396] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:03:34,448] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:03:44,601] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:03:44,668] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:03:44,743] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:03:53,656] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:03:53,761] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:03:53,812] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:04:09,168] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:04:09,292] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:04:09,346] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:04:24,724] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:04:24,914] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:04:24,961] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:05:52,442] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:05:52,546] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:05:52,632] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:06:33,032] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:06:33,239] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:06:33,284] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 11:08:08,856] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 11:08:08,984] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /newsPosts/ HTTP/1.1" 200 173
[2022-08-23 11:08:09,033] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 13:36:01,811] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-08-23 13:36:01,813] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 9499
[2022-08-23 13:55:45,175] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-08-23 13:55:45,176] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 9499
[2022-08-23 14:05:53,047] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-08-23 14:05:53,047] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 9499
[2022-08-23 14:45:15,965] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2022-08-23 14:45:15,966] [basehttp.py:161] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 9499
[2022-08-23 15:10:00,868] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 15:10:02,013] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:10:05,506] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 15:10:06,277] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:10:09,493] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 15:10:10,357] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:11:15,323] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 157207
[2022-08-23 15:11:15,608] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?ordering=-add_Date&limit=20&offset=0 HTTP/1.1" 200 19682
[2022-08-23 15:11:48,734] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 6021
[2022-08-23 15:11:58,302] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/ HTTP/1.1" 200 187670
[2022-08-23 15:12:33,720] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:12:42,603] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:12:44,425] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:12:57,110] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%22%E7%BE%8E%22 HTTP/1.1" 200 187936
[2022-08-23 15:13:00,186] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%22%E7%BE%8E%22 HTTP/1.1" 200 187936
[2022-08-23 15:13:01,336] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%22%E7%BE%8E%22 HTTP/1.1" 200 187936
[2022-08-23 15:13:02,308] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%22%E7%BE%8E%22 HTTP/1.1" 200 187936
[2022-08-23 15:13:03,022] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%22%E7%BE%8E%22 HTTP/1.1" 200 187936
[2022-08-23 15:13:03,996] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%22%E7%BE%8E%22 HTTP/1.1" 200 187936
[2022-08-23 15:13:08,737] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:32,766] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:33,840] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:35,497] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:36,397] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:36,900] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:37,484] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:37,944] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:38,430] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:38,911] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:39,514] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:39,952] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:40,767] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:13:57,992] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/ HTTP/1.1" 200 237923
[2022-08-23 15:14:35,422] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/?fullname=%E5%AE%9D HTTP/1.1" 200 26943
[2022-08-23 15:14:49,825] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/?fullname=%E6%A1%A5 HTTP/1.1" 200 31925
[2022-08-23 15:15:16,838] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/?fullname=%E6%B2%B3 HTTP/1.1" 200 66860
[2022-08-23 15:16:00,368] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/?fullname=&&ordering=-id HTTP/1.1" 200 238939
[2022-08-23 15:16:48,797] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /pack/?fullname=&&ordering=-initiate HTTP/1.1" 200 238981
[2022-08-23 15:17:21,695] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/ HTTP/1.1" 200 187670
[2022-08-23 15:17:42,008] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:17:57,276] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:18:08,841] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:18:27,260] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 15:18:28,378] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:18:30,801] [autoreload.py:251] [autoreload:trigger_reload] [INFO]- E:\zl\git-django-qyzs\django_wx_qhzs\polls\views.py changed, reloading.
[2022-08-23 15:18:31,596] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:18:47,749] [autoreload.py:637] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2022-08-23 15:18:54,681] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:11,316] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:12,107] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:12,767] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:13,200] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:14,199] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:14,715] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:19:15,555] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E HTTP/1.1" 200 187870
[2022-08-23 15:20:21,425] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=%E7%BE%8E&ordering=-id HTTP/1.1" 200 188111
[2022-08-23 15:20:37,892] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=2&ordering=-id HTTP/1.1" 200 188023
[2022-08-23 15:20:46,057] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=7&ordering=-id HTTP/1.1" 200 188023
[2022-08-23 15:20:50,154] [basehttp.py:161] [basehttp:log_message] [INFO]- "GET /news/?title=7&ordering=-id HTTP/1.1" 200 188023
